### ISSUE TITLE
The total number is 13000 of the training datasets, and the input size is [896, 896]. 

### ISSUE THREAD

**Issue by Beam-wi** [xingyizhou/CenterNet#315]

**Comment by Beam-wi** [comment]
The total number is 13000 of the training datasets, and the input size is [896, 896]. I want to know, how big should the learing rate and lr_step is? And how many epoch should i train? I use the hourglass-104.

**Comment by evitself** [comment]
@Beam-wi 
This is an interesting topic, well, I guess there is no standard answer to it. But there is indeed some magics for setting learning rate w.r.t batch size. Actually these magics have already hidden in this centernet implementation. Recall a great paper from R. Girshick and Kaiming.He team in FB, **Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour**, they noticed that people should set lr to **1.25e-2** when batch size is **32**, and scale up/down linearly according to this reference numbers. That is, for example, lr **2.5e-2** for batch size **64**. Although these numbers are tested with SGD on imagenet classification task, but it seems this strategy works well on other task settings, just like here in centernet, with Adam optimizer. A side note, in practice, these numbers are just a reference which helps you start your experiment more quickly, you need to test your specific experiment by yourself to find the optimal hyper-parameters. For example, if you are not going to train your network from scratch, it most likely you don't need a starting learning rate of **1.25e-2** for batch size **32**, but just **1.25e-4** is good for fine-tuning.
Hope this will give you some hints.
A piece of snapshot from the paper mentioned above:
![image](https://user-images.githubusercontent.com/11956064/64075595-b0149d80-ccec-11e9-9970-beeb99db2a6b.png)

**Comment by Beam-wi** [comment]
Thanks for your help!

**Comment by xingyizhou** [comment]
I don't have more specific suggestions than that. Closing it for now.

**Comment by sisrfeng** [comment]
>  For the residual networks and DLA-34, we train with a batch-size of 128 (on 8 GPUs) and learning rate 5**e-4** for 140 epochs.

Would it be better to start with learning rate 5e**-2** when training from strach?

> they noticed that people should set lr to 1.25**e-2** when batch size is 32
